assets/src/edit-story/components/library/panes/media/dropDownMenu.js

```diff
@@ -85,32 +88,42 @@ function DropDownMenu({ pointerEntered, isMenuOpen, setIsMenuOpen }) {
 
   // Keep icon and menu displayed if menu is open (even if user's mouse leaves the area).
   return (
-    (pointerEntered || isMenuOpen) && (
-      <>
-        <MoreButton
-          ref={moreButtonRef}
-          width="28"
-          height="28"
-          onClick={onClickMoreIcon}
-          aria-pressed={isMenuOpen}
-          aria-haspopup={true}
-          aria-expanded={isMenuOpen}
+    <div>
+      {(pointerEntered || isMenuOpen) && (
+        <>
+          <MoreButton
+            ref={moreButtonRef}
+            width="28"
+            height="28"
+            onClick={onClickMoreIcon}
+            aria-pressed={isMenuOpen}
+            aria-haspopup={true}
+            aria-expanded={isMenuOpen}
+          />
+          <Popup anchor={moreButtonRef} isOpen={isMenuOpen}>
+            <DropDownContainer>
+              <DropDownList
+                handleCurrentValue={handleCurrentValue}
+                options={options}
+                toggleOptions={toggleOptions}
+              />
+            </DropDownContainer>
+          </Popup>
+        </>
+      )}
+      {showDeleteDialog && (
+        <DeleteDialog
+          mediaId={mediaId}
+          showDeleteDialog={showDeleteDialog}
+          setShowDeleteDialog={setShowDeleteDialog}
```

Comment:
Perhaps rename this prop (not the value) to `onClose` or similar more generic name.

Reply:
Done.